paper-toolbar.html

```diff
@@ -162,6 +159,36 @@
       pointer-events: auto;
     }
 
+    .toolbar-tools > ::content iron-icon:first-child {
+      margin-right: 32px;
+    }
+
+    .toolbar-tools > ::content paper-icon-button:first-child {
+      margin-left: -8px;
+      margin-right: 24px;
+    }
+
+    .toolbar-tools > ::content paper-icon-button:last-child {
+      margin-right: -8px;
+    }
+
+    .toolbar-tools > ::content iron-icon:not(:first-child):not(:last-child) {
+      margin-right: 8px;
+    }
+
+    .toolbar-tools > ::content [title] {
+      @apply(--paper-font-title);
+      @apply(--layout-flex);
+
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      overflow: hidden;
+    }
+
+    .toolbar-tools > ::content :first-child[title] {
```

Comment:
@frankiefu I have changed the selectors and I confirmed that it's working in shadow/shady mode.